src/worker_pool.rs

```diff
@@ -45,13 +47,52 @@ pub(crate) struct PrecompiledPolicy {
     /// The execution mode of the policy
     pub execution_mode: PolicyExecutionMode,
 }
+lazy_static! {
+    static ref KUBEWARDEN_VERSION: Version = {
+        let mut version = Version::parse(env!("CARGO_PKG_VERSION")).expect("Cannot parse CARGO_PKG_VERSION version");
+        // Remove the patch, prerelease and build information to avoid rejections
+        // like this: v1.6.0-rc4 < v1.6.0
+        version.patch = 0;
+        version.pre = Prerelease::EMPTY;
+        version.build = BuildMetadata::EMPTY;
+        version
+    };
+}
+
+/// Check if policy server version is compatible with  minimum kubewarden
+/// version required by the policy
+fn has_minimum_kubewarden_version(metadata: &Metadata) -> Result<()> {
+    if let Some(minimum_kubewarden_version) = &metadata.minimum_kubewarden_version {
+        let sanitized_minimum_kubewarden_version = Version {
+            major: minimum_kubewarden_version.major,
+            minor: minimum_kubewarden_version.minor,
+            // Kubewarden stack version ignore patch version number
+            patch: 0,
+            // prerelease and build objects does not implement copy trait.
+            // Thus, we need to clone them
```

Comment:
You have to remove this comment, it's outdated. I think you can slightly adapt the one on line 69, because it adapts to patch, pre and build